workloads/files/workload-http-script-cm.yml

```diff
@@ -0,0 +1,1217 @@
+            - labelSelector:
+                matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                  - centos-stress
+              topologyKey: kubernetes.io/hostname
+        containers:
+        - env:
+          - name: IDENTIFIER
+            value: ${IDENTIFIER}
+          - name: NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          envFrom:
+          - configMapRef:
+              name: wlg-env
+          image: quay.io/jmencak/centos-stress
+          imagePullPolicy: Always
+          name: centos-stress
+          resources: {}
+          volumeMounts:
+          - mountPath: /etc/kubernetes/
+            name: k8s-config
+          - mountPath: /opt/wlg/
+            name: targets
+          - mountPath: /opt/stress/.ssh/
+            name: ssh-key
+        hostNetwork: false
+        nodeSelector: "${{NODE_SELECTOR}}"
+        restartPolicy: Never
+        securityContext:
+          sysctls:
+          - name: net.ipv4.ip_local_port_range
+            value: "1024 65535"
+    # "unsafe" sysctl
+    #      - name: net.ipv4.tcp_tw_reuse
+    #        value: "1"
+    #    serviceAccountName: stress
+        tolerations:
+        - effect: NoSchedule
+          key: test
+          operator: Equal
+          value: wlg
+        volumes:
+        - configMap:
+            items:
+            - key: k8s-config
+              path: k8s.conf
+            name: k8s-config
+            optional: true
+          name: k8s-config
+        - configMap:
+            items:
+            - key: wlg-targets
+              path: targets.txt
+            name: wlg-targets
+            optional: true
+          name: targets
+        - name: ssh-key
+          secret:
+            items:
+            - key: wlg-ssh-key
+              path: id_rsa
+            optional: true
+            secretName: wlg-ssh-key
+            # yaml/json doesn't support octal: 400 = (4 * 8^2) + (4 * 8^1) + (4 * 8^0) => 256
+            defaultMode: 256
+    parameters:
+    - description: Number to append to the name of resources
+      name: IDENTIFIER
+      value: '1'
+    - description: nodeSelector definition
+      name: NODE_SELECTOR
+      value: null
+
+  ###################################################################################################
+  server-http.yaml: |
+    apiVersion: v1
+    kind: Template
+    metadata:
+      name: nginx
+    objects:
+    - apiVersion: v1
+      kind: ReplicationController
+      metadata:
+        name: nginx-http-${IDENTIFIER}
+      spec:
+        replicas: 1
+        selector:
+          name: nginx-http-${IDENTIFIER}
+        template:
+          metadata:
+            labels:
+              name: nginx-http-${IDENTIFIER}
+              test: http
+          spec:
+            containers:
+            - image: quay.io/jmencak/nginx
```

Comment:
Like wise to above comment regarding container image I think we should consider allowing this and the other templates with nginx images to be configured.

Reply:
Agreed.